chromium_src/third_party/blink/public/platform/disable_client_hints_browsertest.cc

```diff
@@ -40,7 +42,11 @@ class ClientHintsBrowserTest : public InProcessBrowserTest {
 
   ~ClientHintsBrowserTest() override {}
 
-  void SetUp() override { InProcessBrowserTest::SetUp(); }
+  void SetUp() override {
+    // Test that even with Lang CH feature enabled, there is no header.
+    scoped_feature_list_.InitAndEnableFeature(features::kLangClientHintHeader);
```

Comment:
shouldn't we test with this both enabled and disabled?

Reply:
Addressed by 296d753b902351e5d9653fecae77c58ea267253d